packages/sui-svg/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "devDependencies": {
     "@s-ui/lint": "2",
+    "lodash.camelcase": "4.3.0",
```

Comment:
Maybe use sui-js `https://github.com/SUI-Components/sui/blob/master/packages/sui-js/src/string/index.js#L11` could be a better option, to avoid having to versions of lodas.camelcase in the main bundle.